src/openstudio_lib/RunTabView.cpp

```diff
@@ -231,19 +273,33 @@ void RunView::playButtonClicked(bool t_checked) {
     auto workflowJSONPath = QString::fromStdString(workflowPath.string());
 
     unsigned port = m_runTcpServer->serverPort();
-    bool haveConnection = (port != 0);
+    m_hasSocketConnexion = (port != 0);
+    // NOTE: temp test, uncomment to see fallback to stdout
+    // m_hasSocketConnexion = false;
 
     QStringList arguments;
-    if (haveConnection) {
+    LOG(Debug, "Checkbox is checked? " << std::boolalpha << m_verboseOutputBox->isChecked());
+    if (m_verboseOutputBox->isChecked()) {
+      arguments << "--verbose";
+    } else {
+      // If not verbose, we save the stdout/stderr to a file, like historical
+      // Actually, we don't, we just read it
+      // m_runProcess->setStandardOutputFile(toQString(stdoutPath));
```

Comment:
Will E+ stdout and stderr still be in files in the sim dir for the user to read later?  I see stdout-energyplus and stderr-energyplus files but I'm not entirely sure which process was writing those, the app or the cli.

Reply:
stdout-energyplus stderr-energyplus are written by workflow gem.

What I am no longer outputing is the model_companion_dir/stdout and stderr files. I see no value in keeping those, when we are seeing them live. I don't see any use case for seeing them afterwards, but perhaps there is one. In which case the RunView::readyReadStandardOutput and RunView::readyReadStandardError should be writting to these files in append mode in addition of displaying them live. thoughts @macumber ?

Reply:
Ok yeah, I don't think those two files are useful.  Thanks Julian!

Reply:
cf 4411071f97974765f3d84eee9d272998ad1a17b0